ui/packages/components/src/utils/array.ts

```diff
@@ -0,0 +1,3 @@
+export function isStringArray(value: unknown): value is string[] {
```

Comment:
There's a somewhat similar util in Cloud, if we are doing this one it should replace it: https://github.com/inngest/inngest/blob/main/ui/apps/dashboard/src/utils/isStringArray.ts